site/src/pages/WorkspacePage/WorkspaceReadyPage.tsx

```diff
@@ -100,13 +100,11 @@ export const WorkspaceReadyPage = ({
       ["canceling", "deleting", "pending", "starting", "stopping"].includes(
```

Comment:
This looks like a regression:

<img width="1481" alt="Screenshot 2023-07-18 at 15 40 16" src="https://github.com/coder/coder/assets/14044910/5f3fbca4-86e0-4630-95bb-d9db394c69cf">

Restart form suggests only values, but should suggest defaults.

Reply:
It is not, the function that creates the "initial values" for parameters, is using the values provided from the previous build. I will fix that 👍